Topics/Teamwork.md

```diff
@@ -1,4 +1,26 @@
+Working as a team effectively requires a lot of planning and organization. Projects as a whole are often very complicated and overwhelming, so it is very useful to break down a project into smaller stages/tasks. There are many ways in which a project can be managed; two commons project management practices that software engineering teams user are the Waterfall and Agile methods. Details on these can be found in [Resources for Software Engineers](https://learning-software-engineering.github.io/Topics/Software_Engineering.html#resources-for-software-engineering).
+
+## Team Communication
+During the early stages of forming a team, the group should also establish general expectations for how the team plans to communicate with one another. This may include setting up a team communication channel on a specific platform (ex. discord or slack) and establishing regular check-up meetings (including when these meetings are and where they will be held - for example, in person or online). These general meetings can be used to outline general expectations and project requirements, update one another on individual progress, delegate new reponsibilities, and set project deadlines. 
+
+### Getting Clarification
+You always want to ensure that all team members are on the same page when working together. If you are ever unsure about something that was said during a meeting or have any confusions in general, ask your team members with specific and explicit questions. It is helpful to reiterate your understanding back to the team to confirm your understanding is correct. Some example phrases you may use (taken from [Week 8 Lecture Slide 12](https://q.utoronto.ca/courses/293515/files/25224801/download?download_frd=1)) includes:
+- "Can you please clarify..."
+- "I didn't get the part about..."
+- "What do you mean by..."
+- "How would ... work?"
+- "What are the next steps?"
+
+### Giving and Receiving Feedback 
+When team members are working on different components of a project, it is useful to have meetings where each member can share their work with the rest of the team. During this meeting, constructive feedback can be given to make improvements and offer new insights/perspectives. Feedback should be given early on (for instnace, while an idea is being formulated rather than after it has been fully implemented). Try to be specific when giving feedback by specifying what exactly in the project the feedback is regarding and your opinion is on how it can be improved or what potential issues you may see. In a professional working environment, always ensure feedback is being given with the purpose of supporting the team's goals and improving the project rather than being influenced by any negative emotions. On the other hand, is it good to practice being open and receptive of feedback - understand that everyone has different experiences and opinions and their feedback is not meant to be a criticism but a suggestion that you and the team can benefit from.
```

Comment:
```suggestion
When team members are working on different components of a project, it is useful to have meetings where each member can share their work with the rest of the team. During this meeting, constructive feedback can be given to make improvements and offer new insights/perspectives. Feedback should be given early on (for instance, while an idea is being formulated rather than after it has been fully implemented). Try to be specific when giving feedback by specifying what exactly in the project the feedback is regarding and your opinion on how it can be improved or what potential issues you may see. In a professional working environment, always ensure feedback is being given with the purpose of supporting the team's goals and improving the project rather than being influenced by any negative emotions. On the other hand, it is good to practice being open and receptive to feedback - understand that everyone has different experiences and opinions, and their feedback is not meant to be a criticism but a suggestion that you and the team can benefit from.
```